chsdi/models/clientdata_dynamodb.py

```diff
@@ -42,11 +44,39 @@
 # http://boto.readthedocs.org/en/latest/boto_config_tut.html
 
 
-def get_table(table_name='shorturl'):
+def _get_dynamodb_conn(region='eu-west-1'):
+    try:
+        conn = connect_to_region(region)
+    except Exception as e:
+        raise exc.HTTPBadRequest('DynamoDB: Error during connection init %s' % e)
+    return conn
+
+
+def _get_s3_conn(profile_name='geoadmin_filestorage'):
+    # TODO use profile instead when correctly installed
+    try:
+        conn = connect_s3(profile_name=profile_name)
+    except Exception as e:
+        raise exc.HTTPBadRequest('S3: Error during connection %s' % e)
+    return conn
+
+
+conn_dynamodb = _get_dynamodb_conn()
+conn_s3 = _get_s3_conn()
```

Comment:
this is called at webapp start, correctly? if yes, raising httpbadrequest in the functuons does not really make sense. also, will it stip startup of application?